pkg/server/completion.go

```diff
@@ -163,100 +164,63 @@ func createCompletionItemsFromRanges(ranges []processing.ObjectRange, completion
 	return items
 }
 
-func createCompletionItem(label, prefix string, kind protocol.CompletionItemKind, body ast.Node, position protocol.Position) protocol.CompletionItem {
-	mustNotQuoteLabel := IsValidIdentifier(label)
-
-	insertText := label
-	detail := label
-	if prefix != "" {
-		detail = prefix + "." + insertText
-	}
-	if !mustNotQuoteLabel {
-		insertText = "['" + label + "']"
-		detail = prefix + insertText
-	}
+func formatLabel(str string) string {
+	interStr := "interimPath" + str
+	fmtStr, _ := formatter.Format("", interStr, formatter.DefaultOptions())
```

Comment:
I wonder if we somehow should pass along the formatter options that the LSP already knows of. Also not sure how that would work.